src/components/PathwayExercise/ExerciseContent/Assessment/index.js

```diff
@@ -206,7 +214,7 @@ function Assessment({
           variant="contained"
           sx={{ width: "256px", p: "8px 16px 8px 16px" }}
           color={answer ? "primary" : "secondary"}
-          disabled={answer.length === 0 ? true : false}
+          disabled={answer?.length === 0 ? true : false}
```

Comment:
Agar `answer` `undefined` hai, toh `disabled` ko `true` hona chahiye, na?
```suggestion
          disabled={!answer?.length}
```